docs/conf.py

```diff
@@ -313,8 +321,11 @@ def getmtime(infile):
 
 # The reST default role (used for this markup: `text`) to use for all
 # documents.
-#
-# default_role = None
+# Imported from sphinx_astropy
+# default_role = 'obj'
```

Comment:
The PR describes as you setting this value, but it's commented out here. I'm confused now!

Reply:
I'm indirectly setting it by importing it from sphinx_astropy: https://github.com/sherpa/sherpa/pull/1268/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R35

I'm doing that because I think there is more that we can import in general (e.g. the graviz-dot settings), so we don't have to maintain the settings should they change, but I also don't just want to import everything at this point (that messes up our html style).

Reply:
Aha. The `v1` on that import worries me slightly - is there already a `v2` (in other words are we tying ourself to code that we willhave to change once astropy decides `v1` is so old it can be removed)?

Reply:
There is no `v2` yet (and there might never be). The whole point of having this versioned is that `v1` can stay around forever and it's up to the projects that use it to consciously switch to `v2` should it ever come. Astropy uses a similar scheme in other area (e.g. your choice of cosmological parameters) and to my knowledge none of the old version has ever been removed since the inception of the project.

Reply:
Ta - just wanted to know what the strategy was.